content/SCALE/SCALETutorials/Shares/MixedModeShares.md

```diff
@@ -0,0 +1,238 @@
+
+If Active Directory is already joined to the TrueNAS server, click **Save** and then reopen the **NFS** configuration screen.
+Click **Add SPN** to open the **Add Kerberos SPN Entry** dialog.
+
+{{< trueimage src="/images/SCALE/SystemSettings/ServicesNFSAddKerberosSPNEntry.png" alt="Add Kerberos SPN Entry" id="Add Kerberos SPN Entry" >}}
+
+Click **Yes** when prompted to add a Service Principal Name (SPN) entry.
+Enter the AD domain administrator user name and password in **Name** and **Password**.
+
+{{< hint type=tip >}}
+TrueNAS SCALE automatically applies SPN credentials if the NFS service is enabled with **Require Kerberos for NFSv4** selected before joining Active Directory.
+{{< /hint >}}
+
+### Joining Active Directory
+
+Mixed-mode SMB and NFS shares greatly simplify data access for client running a range of operating systems.
+They also require careful attention to security complexities not present in standard SMB shares.
+NFS shares do not respect permissions set in the SMB Share ACL.
+The NFS admin should protect the NFS export with proper authentication and authorization controls to prevent unauthorized access by NFS clients.
+
+We recommend using [Active Directory]({{< relref "configadscale.md" >}}) to enable Kerberos security for the NFS share.
+Configure a container (group or organizational unit), Kerberos admin, and user accounts in AD.
+
+### Creating a Multiprotocol Share Dataset
+
+Before creating a mixed-mode share, create the dataset you want the share to use for data storage.
+
+{{< include file="/content/_includes/ShareDatasetsNotPools.md" >}}
+
+We recommend creating a new dataset with the **Share Type** set to **Multiprotocol** for the new mixed-mode share.
+
+{{< expand "Creating a Dataset" "v" >}}
+{{< include file="/content/_includes/CreateDatasetSCALE.md" >}}
+{{< /expand >}}
+
+#### Adjusting the Dataset ACL
+
+After joining AD and creating a dataset, you need to adjust the dataset/filesystem ACL to match the container and users configured in AD.
+
+{{< expand "Click here for instructions" "v" >}}
+1. Go to **Datasets**.
+
+2. Click on the name of the dataset you created for the multiprotocol share to use.
+
+3. Scroll down to the **Permissions** widget. Click **Edit** to open the **Edit ACL** screen.
+
+4. Check the **Access Control List** to see if the AD group you created is on the list and has the correct permissions. If not, add this Access Control Entry (ACE) item.
+
+   a. Enter **Group** in the **Who** field or use the dropdown list to select **Group**.
+
+   b. Type or select the appropriate group in the **Group** field.
+
+   c. Verify **Full Control** displays in **Permissions**. If not, select it from the dropdown list.
+
+   d. Click **Save Access Control List** to add the ACE item or save changes.
+
+See [Permissions]({{< relref "PermissionsScale.md" >}}) for more information on editing dataset permissions.
+{{< /expand >}}
+
+### Creating the SMB Share
```

Comment:
is there a reason why you can't link to the existing procedure instead of rewriting the instructions here? If the only thing that differs is the dataset you select, you can say this earlier in the overview procedure.

Reply:
Again, I think there's value in having a single cohesive procedure here rather than linking out. Every step has some unique variable and I think as a reader I'd rather have a continuous process than constantly have to read another article but also remember the specific ways in which my application differs from the instructions in that other article.